entries/b/bokio.se.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Bokio Sverige": {
+    "domain": "bokio.se",
+    "url": "https://www.bokio.se",
+    "img": "bokio.svg",
```

Comment:
Please rename the image file to `bokio.se.svg` and remove the `img` element as it is unnecessary.
```
```